### PR TITLE
Update to 3.0.0 API and scheduler changes

### PR DESCRIPTION
**Pull request by Aericio**
https://github.com/pmmp/PocketMine-MP/commit/9644766df3c69ec52b5bd3058776b0151fac488d

* bumps version to 2.3.3
* updates for scheduler
* 3.0.0 api

### CODE REVIEW

**Comment by SuperAdam47** [comment]
error @Aericio please fix
screenshot_20180615-094042

**Comment by Aericio** [comment]
Not caused by this commit.

**Comment by Aericio** [comment]
Refer to https://github.com/SalmonDE/TopVoter/issues/23 for your issue.

File: plugin.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: TopVoter
-version: 2.3.2
-api: [3.0.0-ALPHA9, 3.0.0-ALPHA10, 3.0.0-ALPHA11, 3.0.0-ALPHA12]
+version: 2.3.3
```
Comment (SalmonDE) [inline plugin.yml]: minor bump, please. -> 2.4.0

**Review by SalmonDE — APPROVED** [review]
Looks good to me, just bump the minor version.

**Pull request merged**